sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -254,6 +251,21 @@ class SparkSqlAstBuilder extends AstBuilder {
     }
   }
 
+   /**
+    * A column path can be specified as an parameter to describe command. It is a dot separated
+    * elements where the last element can be a String.
+    * TODO - check with Herman
```

Comment:
Yeah Herman. Not supporting it would certainly simplify things. FYI -  I checked that the unit test case describe_xpath.q which exercises this syntax is not getting tested in HiveCompatibleSuite.